packages/advanced-logic/test/utils/payment-network/erc777/stream-create-data-generator.ts

```diff
@@ -160,7 +143,7 @@ const baseRequestState = {
     value: TestData.payeeRaw.address,
   },
   currency: {
-    network: network,
+    network: RINKEBY,
```

Comment:
by keeping the variable name network you can remove : network as Alex suggested it for version.
So that's another argument to keep naming it network.